docs/community/intro.md

```diff
@@ -4,12 +4,28 @@ sidebar_label: 🧐 How It Works
 
-These contributions are connected to each other based on their relationships. For example, a pull request may fix a particular bug, and a code review may review that pull request.
+Anything a contributor does to support a project they're working on - from the technical (like writing code or filing bug reports), to the logistical (like organizing events or work sessions), to the emotional (like mediating difficult discussions or helping someone feel welcome) is considered a contribution that has some amount of value. 
 
-This network of contributions and relationships form a contribution graph. SourceCred applies the [PageRank algorithm](https://en.wikipedia.org/wiki/PageRank)  to this graph, generating a score, called cred for every contribution.
+Each of these contributions made to the project are connected to each other in a large graph of "nodes" (the contributions and contributors) and "edges" (the lines that connect nodes together). This graph shows the way each contribution and contributor in the project relates to the other nodes it affects or supports. 
 
-The basic idea is: a contribution earns cred if it is connected to other contributions that earn lots of cred. Contributors are connected to the contributions they've helped with, which means that they also earn cred.
+This large graph of every contribution made to a project and the ways those contributions are connected is then analized by a [PageRank algorithm](https://en.wikipedia.org/wiki/PageRank) which generates a score for each contribution and contributor based on how connected they are to the rest of the graph. The more valuable the contribution is, the higher it's score. 
 
-The project's community and maintainers have a lot of influence over this process. They can reconfigure SourceCred weights and parameters. For example, they could assign extra weight to some very important contributions, or decrease the weights of contributions that seem spammy. They can also add heuristics to do this automatically.
+These value scores are called "Cred". 
+
+So how does the algorithm determine how valuable a contribution is, and therefore how much Cred should it get? A contribution shows that it's valuable and earns Cred when it is connected to other valuable contributions in the graph. 
+
+For example, say you have a meeting and during that meeting you come up with an incredible idea. That idea eventually turns into a new plugin or feature that really moves the project forward. All the concrete results of your idea will connect through the graph back to that one meeting where it was first dreamt up. Because the node that represents your idea is connected to all the contributions that came out of it, it will end up with a lot of Cred. And so will you, as the person who had that original idea, because contributors are connected to the contributions they've created or helped with. 
```

Comment:
This would be a good time to include the diagram that you drew--@hammadj and I can help get it included.